README.md

```diff
@@ -41,6 +56,9 @@ steps:
 | - | - | - |
 | `tests` | files or directories containing the MOxUnit test cases | *Optional*, defaults to the root directory of the repo. All subdirectories are added recursively. |
 | `src` | directories to be added to the Octave search path before running the tests. These directories will be considered in the coverage reports, if coverage is enabled. | *Optional*
+| `data` | directory for test data | *Optional* |
+| `pkg` | Octave packages to load. Available options: `image` `io` `optim` `parallel` `statistics` `struct`| *Optional* |
+| `pkg` | External resources to add to the search put (excluded from coverage)| *Optional* |
```

Comment:
I just found a typo: This should be `ext` and not `pkg`. I can add it myself, but maybe you want to fix this in your fork as well, @agahkarakuzu